pallets/schemas/README.md

```diff
@@ -4,97 +4,170 @@ The Schemas Pallet provides universal schema registration for data flowing throu
+  locations)
 
 #### Settings
 
-- [`AppendOnly`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.SchemaSetting.html#variant.AppendOnly)
-  - Prior data is immutable and all new data is appended to existing data.
-  - For Payload Locations: `Itemized`
-- [`SignatureRequired`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.SchemaSetting.html#variant.SignatureRequired)
-  - An MSA control key signature is required instead of a delegation.
-  - For Payload Locations: `Itemized` or `Paginated`
+- [
+  `AppendOnly`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.SchemaSetting.html#variant.AppendOnly)
+    - Prior data is immutable and all new data is appended to existing data.
+    - For Payload Locations: `Itemized`
+- [
+  `SignatureRequired`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.SchemaSetting.html#variant.SignatureRequired)
+    - An MSA control key signature is required instead of a delegation.
+    - For Payload Locations: `Itemized` or `Paginated`
 
 #### Payload Locations
 
-- [`OnChain`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.OnChain): Data is stored directly in the Messages pallet data storage, usually as `AvroBinary`.
-- [`IPFS`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS): Data is stored in IPFS and Messages pallet stores the CID.
-- [`Itemized`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.Itemized): Data is stored in the Stateful Storage pallet as an array of individual items.
-- [`Paginated`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.Paginated): Data is stored in the Stateful Storage pallet as a list of paged blobs.
-
-### Mainnet vs Testnet Schema Creation
-
-Mainnet schemas must be approved by the Frequency Council.
+- [
+  `OnChain`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.OnChain):
+  Data is stored directly in the Messages pallet data storage, usually as `AvroBinary`.
+- [`IPFS`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS):
+  Data is stored in IPFS and Messages pallet stores the CID.
+- [
+  `Itemized`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.Itemized):
+  Data is stored in the Stateful Storage pallet as an array of individual items.
+- [
+  `Paginated`](https://frequency-chain.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.Paginated):
+  Data is stored in the Stateful Storage pallet as a list of paged blobs.
+
+### Mainnet vs Testnet Entity Creation
+
+Mainnet Schemas, Intents, and Intent Groups must be approved by the Frequency Council.
```

Comment:
I think it would be useful to summarize a little more specifically by saying on mainnet, those wishing to create schemas, intents and intent groups must call the `propose_to_create...` extrinsics, whereas on testnet they can simply call the `create_...` extrinsics directly.  Also that the `create_foo_via_governance` extrinsics aren't generally meant to be called directly.